org.ekstep.questionunit.ftb-1.0/renderer/styles/style.css

```diff
@@ -5,7 +5,7 @@
   width: 80%;
   margin-top: 2%;
   /* display: grid; */
-  height: 90vh;
+  height: 80%;
```

Comment:
@rajeshcompass did you verified in Mobile and web with these changes?

Reply:
yes , i verified in web, mobile and tab